app/src/main/java/com/poupa/vinylmusicplayer/dialogs/BottomSheetDialog/EnqueueSongsBottomSheetDialog.java

```diff
@@ -0,0 +1,36 @@
+package com.poupa.vinylmusicplayer.dialogs.BottomSheetDialog;
+
+
+import java.util.List;
+
+import com.poupa.vinylmusicplayer.util.PreferenceUtil;
+
+
+public class EnqueueSongsBottomSheetDialog extends ButtonListBottomSheetDialog {
+    public static EnqueueSongsBottomSheetDialog newInstance() { return new EnqueueSongsBottomSheetDialog(); }
+
+    int defaultChoice;
+
+    public EnqueueSongsBottomSheetDialog setDefaultChoice(int defaultIndex) {
+        this.defaultChoice = defaultIndex;
+        return this;
+    }
+
+    @Override
+    public EnqueueSongsBottomSheetDialog setTitle(String title) {
+        super.setTitle(title);
+        return this;
+    }
+
+    @Override
+    public EnqueueSongsBottomSheetDialog setButtonList(List<Item> buttonList) {
+        super.setButtonList(buttonList);
+        return this;
+    }
+
+    void afterClickOn(int i) {
+        defaultChoice = i;
+        PreferenceUtil.getInstance().setEnqueueSongsDefaultChoice(this.defaultChoice);
```

Comment:
As discussed, we dont need to save and restore the default choice

Reply:
> Yes I was keeping the code just in case we wanted to add a option to no always show the menu (as it is not that big of dead code)

So we don't want to keep this just in case ?

Reply:
I'd be aggressive and drop it now, before a release is made (otherwise it will become legacy)